webpack/ForemanTasks/Components/TasksTable/TasksTable.js

```diff
@@ -92,19 +93,19 @@ const TasksTable = ({
     updateURlQuery({ sort_by: by, sort_order: order }, history);
   };
 
-  const taskActions = {
+  const modalActions = {
```

Comment:
why do we need the rename of taskActions all over the PR actually?
I think it make more sense to have taskActions which we insert into a specific modal.
because we can have other modals where this actions doesn't really fit.

Reply:
I started to get confused with the other var named tasksaction from here:
https://github.com/theforeman/foreman-tasks/blob/3faffd08f7c8f5b36f92f574c6283cb6c12373a0/webpack/ForemanTasks/Components/TasksTable/TasksTablePage.js#L65
As modalActions had actions that open modals and tasks actions had just task actions

Reply:
yes, it is quite confusing.. since also `tasksActions` contains modal functions.
I would suggest to pass each individual prop to the `ConfirmationModals`, e.g:
```js
<ConfirmationModals
       cancelTask={cancelTask}
       resumeTask={resumeTask}
       cancelSelectedTasks={cancelSelectedTasks}
       resumeSelectedTasks={resumeSelectedTasks}
/>
```

Reply:
When the force cancel will be added I think it's might be too much (6 actions total), and the `resumeTask` will be something like `resumeTaskWithID` which looks to me not that organized

Reply:
I don't see problem with having many props, I prefer that instead of passing a huge object.

btw, we can always create more generic actions which will handle regular or bulk function, based on the amount of IDs that was provided, right?

Reply:
I think i generalized them as much as I could, do you have a suggestion to make them more general?

Reply:
Also the consts in `tasksActions` can be used comfortably when choosing  which function to use for which modal, and moving it to props will cause an (in my opinion) unnecessary switch case block

Reply:
We could investigate about it more in another PR though meanwhile I think passing each func as a prop would be great.

Reply:
Yeah but instead of passing the object there would have to be a convertion to 4 consts to pass as props, and then a switch case to choose which prop to use:
```JS 
TasksTablePage.js
const cancelTask={
...
};
const resumeTask={
...
};
const cancelSelectedTasks={
...
};
const resumeSelectedTasks={
...
};

CofirmationModals.js
const
const getFunc = type =>{
  switch (type) {
    case CANCEL_MODAL:
      cancelTask;
    case RESUME_MODAL:
      resumeTask;
    case CANCEL_SELECTED_MODAL:
      cancelSelectedTasks;
    case RESUME_SELECTED_MODAL:
      resumeSelectedTasks;
  }  
}
```
Why not to just pass an object instead?

Reply:
lets continue the discussion to the right line,
as for this specific line in the TaskTable I guess it could be renamed to `modalActions` as you did, because it uses `createTasksTableSchema`